MAUI/DataGrid/Styling.md

```diff
@@ -234,37 +266,45 @@ You can apply the style for header and row font attributes by using SfDataGrid.D
 
 {% tabs %}
 {% highlight xaml tabtitle="MainPage.xaml"%}
-<syncfusion:SfDataGrid ItemsSource="{Binding OrderInfoCollection}">
-    <syncfusion:SfDataGrid.DefaultStyle>
-        <syncfusion:DataGridStyle HeaderRowFontAttributes="Bold"
-                                  HeaderRowFontFamily="TimesNewRoman"
-                                  HeaderRowFontSize="16"
-                                  RowFontAttributes="Italic"
-                                  RowFontFamily="Adabi"
-                                  RowFontSize="14"/>
-    </syncfusion:SfDataGrid.DefaultStyle>
-</syncfusion:SfDataGrid>
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Content>
+        <syncfusion:SfDataGrid ItemsSource="{Binding OrderInfoCollection}">
+            <syncfusion:SfDataGrid.DefaultStyle>
+                <syncfusion:DataGridStyle HeaderRowFontAttributes="Bold"
+                                        HeaderRowFontFamily="TimesNewRoman"
+                                        HeaderRowFontSize="16"
+                                        RowFontAttributes="Italic"
+                                        RowFontFamily="Adabi"
+                                        RowFontSize="14"/>
+            </syncfusion:SfDataGrid.DefaultStyle>
+        </syncfusion:SfDataGrid>
+    </ContentPage.Content>
+</ContentPage>
 {% endhighlight %}
 {% endtabs %}
 
 ![Changing the font style .NET MAUI DataGrid](Images/styling/maui-datagrid-font-style.png)
 
 ## Border customization
-The data grid customizes the grid borders to vertical, horizontal, both, or none. Set desired value to GridLinesVisibility property in SfDataGrid.GridLinesVisibility or SfDataGrid.HeaderGridLinesVisibility
+The data grid customizes the grid borders to vertical, horizontal, both or none. Set desired value to [SfDataGrid.GridLinesVisibility] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_GridLinesVisibilityProperty) or [SfDataGrid.HeaderGridLinesVisibility] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_HeaderGridLinesVisibilityProperty)
 Following are the list of options available to customize the grid borders:
 .	Both
 .	Horizontal
 .	Vertical
 .	None
 
 ### Both
-The GridLinesVisibility.Both displays the data grid with both horizontal and vertical borders.
+The [GridLinesVisibility.Both] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.GridLinesVisibility.html#Syncfusion_Maui_DataGrid_GridLinesVisibility_Both) displays the data grid with both horizontal and vertical borders.
```

Comment:
The [GridLinesVisibility.Both] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.GridLinesVisibility.html#Syncfusion_Maui_DataGrid_GridLinesVisibility_Both) display the data grid with both horizontal and vertical borders.